pkg/operation/botanist/waiter.go

```diff
@@ -37,7 +37,7 @@ import (
 func (b *Botanist) WaitUntilKubeAPIServerServiceIsReady() error {
 	var e error
 	if err := wait.Poll(5*time.Second, 600*time.Second, func() (bool, error) {
-		loadBalancerIngress, err := common.GetLoadBalancerIngress(b.K8sSeedClient, b.Shoot.SeedNamespace, common.KubeAPIServerDeploymentName)
+		loadBalancerIngress, err := common.GetLoadBalancerIngress(context.TODO(), b.K8sSeedClient.Client(), b.Shoot.SeedNamespace, common.KubeAPIServerDeploymentName)
```

Comment:
Make `WaitUntilKubeAPIServerServiceIsReady` '`context`ified and pass the outer context in, also applying the timeout to it.

Reply:
done